ui/src/app/views/workspace-nav-bar/component.spec.tsx

```diff
@@ -0,0 +1,68 @@
+import {currentWorkspaceStore, NavStore} from 'app/utils/navigation';
+import {WorkspaceNavBarReact} from 'app/views/workspace-nav-bar/component';
+import {mount} from 'enzyme';
+import {WorkspaceAccessLevel} from 'generated';
+import * as React from 'react';
+import {WorkspacesServiceStub} from 'testing/stubs/workspace-service-stub';
+
+describe('WorkspaceNavBarComponent', () => {
+
+  let props: {};
+  const workspace = {
+    ...WorkspacesServiceStub.stubWorkspace(),
+    accessLevel: WorkspaceAccessLevel.OWNER,
+  };
+
+  const component = () => {
+    return mount(<WorkspaceNavBarReact {...props}/>, {attachTo: document.getElementById('root')});
```

Comment:
This `attachTo` makes the PopupTrigger work.

Reply:
awesome! good find